components/cover/tormatic.rst

```diff
@@ -0,0 +1,101 @@
+Tormatic/Novoferm Cover
+=======================
+
+.. seo::
+    :description: Setting up Tormatic and Novoferm covers in ESPHome.
+    :image: tormatic.png
+
+The ``tormatic`` cover platform allows you to control Tormatic and Novoferm
+garage door drives manufactured in 2016 onwards. The following models should
+be supported:
+
+ * Tormatic Black 600, Black 800, Black 1000
+ * Novoferm Novomatic 423
+ * Novoferm Novomatic 563 S
+ * Novoferm Novoport IV
+
+.. figure:: images/tormatic.png
+    :align: center
+
+    Tormatic Black 600 Garage Door Drive
+
+.. note::
+
+    This component was written and tested against a Novoferm 423, so
+    compatibility with other models is currently not guaranteed. Please update
+    this documentation if you confirm this to work on other models as well.
+    
+    The vendor's cloud-based remote is not known to perform model or feature
+    detection, so it's assumed that the protocol is the same across all models.
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    uart:
+      rx_pin: GPIO19
+      tx_pin: GPIO20
+      baud_rate: 9600
```

Comment:
We do not add other components into examples, but add a sentence above specifying that `uart` is required with a link to the uart docs. You can state the baud rate that is required etc at the same time.

```suggestion
    # Example configuration entry
```

Reply:
> We do not add other components into examples

That's surprising, because I borrowed this block from another component. [Here](https://esphome.io/components/sensor/mhz19) [are](https://esphome.io/components/sensor/hm3301) [a](https://esphome.io/components/sensor/ccs811) [few](https://esphome.io/components/sensor/senseair) [more](https://esphome.io/components/sensor/sm300d2) [examples](https://esphome.io/components/sensor/t6615).

Looking around, many components do indeed link to the UART docs, I'll add that as well.